sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/UserPrincipal.java

```diff
@@ -100,5 +100,13 @@ public String getSubject() {
     public Object getClaim(String name) {
         return jwtClaimsSet == null ? null : jwtClaimsSet.getClaim(name);
     }
+
+    public String getName() {
+        return jwtClaimsSet == null ? null : (String) jwtClaimsSet.getClaim("name");
+    }
+
+    public String getUserPrincipalName() {
```

Comment:
Should we use `getUserPrincipalName` or `GetPerferredUsername` here?

Reply:
I am not sure, in portal.azure.com, it is called “User Principal Name”  
![image](https://user-images.githubusercontent.com/4465723/98188914-dec08080-1f4e-11eb-9236-69659e0774ee.png)